src/containers/RecordingView.tsx

```diff
@@ -40,22 +40,23 @@ type Props = StateProps & DispatchProps & OwnProps & RouteComponentProps
 const RecordingView: FunctionComponent<Props> = ({ recordingId, recording, sortings, history, documentInfo, onSetRecordingInfo, plugins }) => {
   const hither = useContext(HitherContext)
   const { documentId, feedUri, readOnly } = documentInfo;
+  const [recordingInfo, setRecordingInfo] = useState<RecordingInfo | null>(null)
+  const [errorMessage, setErrorMessage] = useState<string | null>(null)
 
-  const effect = async () => {
-    if (!recording) return;
-    const rec = recording;
-    if (!rec.recordingInfo) {
-      try {
-        const info = await getRecordingInfo({ recordingObject: rec.recordingObject, hither });
-        onSetRecordingInfo({ recordingId: rec.recordingId, recordingInfo: info });
-      }
-      catch (err) {
-        console.error(err);
-        return;
+  useEffect(() => {
```

Comment:
Cleaned up this effect. It should be improved more in the future by centralizing the getRecordingInfo via useRecordingInfo custom hook. Making an issue for this.